src/client/common/application/commandManager.ts

```diff
@@ -5,9 +5,12 @@
 
 import { injectable } from 'inversify';
 import { commands, Disposable, TextEditor, TextEditorEdit } from 'vscode';
+import { Telemetry } from '../../datascience/constants';
+import { sendTelemetryEvent } from '../../telemetry';
 import { ICommandNameArgumentTypeMapping } from './commands';
 import { ICommandManager } from './types';
 
+const commandsToIgnore = new Set<string>(['setContext']);
```

Comment:
Could we rename this variable to capture the fact that it's telemetry specific or add a comment to that effect?

Reply:
Done